src/app/Dashboard/LayoutTemplateGroup.tsx

```diff
@@ -0,0 +1,242 @@
+  SerialCardConfig,
+} from './dashboard-utils';
+
+export interface LayoutTemplateGroupProps {
+  title: string;
+  templates: LayoutTemplate[];
+  onTemplateSelect: (templateName: LayoutTemplate) => void;
+  onTemplateDelete: (templateName: string) => void;
+}
+
+export const smallestFeatureLevel = (cards: SerialCardConfig[]): FeatureLevel => {
+  return cards.reduce((minFeatureLevel: FeatureLevel, card: CardConfig) => {
+    const featureLevel = getConfigByName(card.name).featureLevel;
+    return Math.min(minFeatureLevel, featureLevel);
+  }, FeatureLevel.PRODUCTION);
+};
+
+export const LayoutTemplateGroup: React.FC<LayoutTemplateGroupProps> = ({
+  onTemplateSelect,
+  onTemplateDelete,
+  ...props
+}) => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const { selectedTemplate, setSelectedTemplate } = React.useContext(LayoutTemplateContext);
+  const scrollRef = React.useRef<HTMLDivElement>(null);
+
+  const handleTemplateSelect = React.useCallback(
+    (template: LayoutTemplate) => {
+      setSelectedTemplate(template);
+      onTemplateSelect(template);
+    },
+    [setSelectedTemplate, onTemplateSelect]
+  );
+
+  const handleClearRecent = React.useCallback(() => {
+    dispatch(dashboardConfigTemplateHistoryClearIntent());
+  }, [dispatch]);
+
+  return (
+    <>
+      <Split>
+        <SplitItem>
+          <Title headingLevel="h2" size="lg" style={{ padding: '1em' }}>
+            {props.title} ({t('LayoutTemplateGroup.ITEMS', { count: props.templates.length })})
+          </Title>
+        </SplitItem>
+        {props.title === t('SUGGESTED', { ns: 'common' }) && props.templates.length !== 1 && (
+          <>
+            <SplitItem isFilled></SplitItem>
+            <SplitItem>
+              <Button variant="link" onClick={handleClearRecent}>
+                {t('CLEAR_RECENT', { ns: 'common' })}
+              </Button>
+            </SplitItem>
+          </>
+        )}
+      </Split>
+      <Gallery hasGutter className="layout-template-picker">
+        {props.templates.map((template) => {
+          const level = smallestFeatureLevel(template.cards);
+          return (
+            <FeatureFlag key={template.name} level={level}>
+              <div
+                ref={scrollRef}
+                className={
+                  selectedTemplate?.name === template.name && selectedTemplate.vendor == template.vendor
+                    ? 'layout-template-card__featured'
+                    : undefined
+                }
+              >
+                <CatalogTile
+                  featured={selectedTemplate?.name === template.name && selectedTemplate.vendor == template.vendor}
+                  id={template.name}
+                  key={template.name}
+                  icon={iconify(template.vendor)}
+                  title={template.name}
+                  vendor={template.vendor}
+                  onClick={() => handleTemplateSelect(template)}
+                  badges={[
```

Comment:
Is it possible to put the labels under or next to title? Or is it designed this way already?

![Screenshot from 2023-04-12 22-05-46](https://user-images.githubusercontent.com/68053619/231628207-74bc0d2d-5e94-47b6-8018-6106c1ae4313.png)